ample/modelling/rosetta_model.py

```diff
@@ -611,8 +609,10 @@ def mr_cmd(self,template,alignment,nstruct,seed):
         return cmd
     
     def nmr_remodel(self, models, ntimes=None, alignment_file=None, remodel_fasta=None, monitor=None):
-        if remodel_fasta: assert os.path.isfile(remodel_fasta), "Cannot find remodel_fasta: {0}".format(remodel_fasta)
-        if ntimes: assert type(ntimes) is int, "ntimes is not an int: {0}".format(ntimes)
+        if remodel_fasta:
+            assert(os.path.isfile(remodel_fasta), "Cannot find remodel_fasta: {0}".format(remodel_fasta))
+        if ntimes:
+            assert(type(ntimes) is int, "ntimes is not an int: {0}".format(ntimes))
```

Comment:
This could also be replaced with:
`assert isinstance(ntimes, int), "ntimes is not an int: {0}".format(ntimes)`